src/code/z_horse.c

```diff
@@ -1,7 +1,15 @@
 #include "global.h"
 #include "vt.h"
 
-s32 func_8006CFC0(s32 scene) {
+#define HORSETYPE_1 1
+#define HORSETYPE_2 2
+#define HORSETYPE_5 5
+#define HORSETYPE_6 6
+#define HORSETYPE_7 7
+#define HORSETYPE_8 8
+#define HORSETYPE_9 9
```

Comment:
These should probably be an enum, maybe inside z_en_horse.h since they're used for the EnHorse actor's params (`Actor#params`, a s16 value that is usually used to set actor properties, eg speed for a moving platform, or the contents of a chest)

usually we name enum types `SystemNameWhatever` like for example `DemoKekkaiType` https://github.com/zeldaret/oot/blob/master/src/overlays/actors/ovl_Demo_Kekkai/z_demo_kekkai.h#L32
I would name the enum values themselves like `ENHORSE_TYPE_*` but if you feel like `HORSE_TYPE_*` or `HORSE_*` or something else is better I personally don't mind

also, you can find in z_en_horse by looking at params usage that there's also `HORSETYPE_3` (used in `EnHorseGameCheck_InitIngoRace` afaict from searching for where `ACTOR_EN_HORSE` is used)

also z_en_horse.c should use these values as well when checking params

Reply:
These horse types are used in multiple files. 
How do we handle this, eg. where would we put such an enum? 
Maybe put that enum inside of `z_horse.h` and `#include "z_horse.h"` in `z_en_horse.c`??
Or maybe create a completely separate header for that enum? (I am relatively new to c programming, I have done mostly oop in c++ and java)

Reply:
the header file for the horse actor (z_en_horse) would make most sense. 
You picked an unusual file as your first to document, in that its a file in the "engine" (so to speak) with code relating to a specific actor. So even though this code isnt the actor itself, it helps facilitate things for that actor.
Player is the other main example of this, with the actor being z_player.c and the helper stuff being z_player_lib.c

Reply:
As fig said, anything used by a specific actor should go in that actor's header file

Then files needing to interact with that actor include the header

For example z_camera includes z_en_horse.h https://github.com/zeldaret/oot/blob/0b8edc21c03a719d8f7874bc20204612d6d1c107/src/code/z_camera.c#L4
to use for example the `EN_HORSE_CHECK_JUMPING` macro defined in the EnHorse actor's header